pkg/master/controller.go

```diff
@@ -42,10 +43,14 @@ import (
 // loops, which manage creating the "kubernetes" service, the "default" and "kube-system"
 // namespace, and provide the IP repair check on service IPs
 type Controller struct {
+	runner *util.Runner
+
 	NamespaceRegistry namespace.Registry
 	ServiceRegistry   service.Registry
-	// TODO: MasterCount is yucky
+	// MasterCount is superceded by MasterLeases
```

Comment:
I think we have different commenting styles, you might not believe it from how I speak IRL, but I like comments to be serious and dry :)